README.md

```diff
@@ -71,27 +69,34 @@ Once this is done, you can go ahead and install prometheus-service:
 
 * Install Keptn prometheus-service in Kubernetes using
 
-```bash
-helm upgrade --install -n keptn prometheus-service https://github.com/keptn-contrib/prometheus-service/releases/download/<VERSION>/prometheus-service-<VERSION>.tgz --reuse-values
-```
-
-* Install Role and RoleBinding to permit prometheus-service for performing operations in the Prometheus `monitoring` namespace:
-
-```bash
-kubectl -n monitoring apply -f https://raw.githubusercontent.com/keptn-contrib/prometheus-service/<VERSION>/deploy/role.yaml
-```
-
-* (Optional) Replace the environment variable value according to the use case and apply the manifest:
+    ```bash
```

Comment:
Maybe specify somewhere that the command will assume `keptn` and `monitoring` as default namespaces if not already set in a previous install...